src/js/chromecast/ChromecastSessionManager.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+var Class = require('class.extend'),
+    _ = require('underscore'),
+    hasConnected = false, // See the `isChromecastConnected` function.
```

Comment:
Discussed outside of this PR: existing comments are sufficient to explain why a static variable is necessary.